packages/wrangler/src/__tests__/d1/export.test.ts

```diff
@@ -14,19 +14,90 @@ describe("execute", () => {
 	mockApiToken();
 	mockConsoleMethods();
 	runInTempDir();
-	const { mockOAuthServerCallback } = mockOAuthFlow();
 	const { setIsTTY } = useMockIsTTY();
+
 	it("should throw if output is missing", async () => {
 		await expect(runWrangler("d1 export db --local")).rejects.toThrowError(
 			`Missing required argument: output`
 		);
 	});
 
-	it("should reject --local mode (for now)", async () => {
-		await expect(
-			runWrangler("d1 export db --local --output /tmp/test.sql")
-		).rejects.toThrowError(
-			`Local imports/exports will be coming in a future version of Wrangler.`
+	it("should handle local", async () => {
```

Comment:
You should probably add tests for the edge cases like quoted names adding the "IF NOT EXISTS" as well as properly quoting things like keywords.

Reply:
I have all these inside the D1 codebase, but see https://github.com/cloudflare/workers-sdk/pull/6073#discussion_r1655712242 I think it makes sense for this file to live in a separate package with a whole suite of tests (as well as a test runner that runs native SQLite for comparison like the D1 suite does).

Imo that's a follow-up task. For the moment I'm just keeping the two implementations in sync manually.

Reply:
I agree it could be a follow-up task.